Traceback (most recent call last):
  File "/home/yichia3/torchgeo/train.py", line 153, in <module>
    pl.seed_everything(conf.program.seed)
  File "/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/lightning/fabric/utilities/seed.py", line 47, in seed_everything
    seed = int(seed)
ValueError: invalid literal for int() with base 10: '1, 2'
Traceback (most recent call last):
  File "/home/yichia3/torchgeo/train.py", line 153, in <module>
    pl.seed_everything(conf.program.seed)
  File "/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/lightning/fabric/utilities/seed.py", line 47, in seed_everything
    seed = int(seed)
ValueError: invalid literal for int() with base 10: '1, 2'
Traceback (most recent call last):
  File "/home/yichia3/torchgeo/train.py", line 153, in <module>
    pl.seed_everything(conf.program.seed)
  File "/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/lightning/fabric/utilities/seed.py", line 47, in seed_everything
    seed = int(seed)
ValueError: invalid literal for int() with base 10: '1, 2'
Traceback (most recent call last):
  File "/home/yichia3/torchgeo/train.py", line 153, in <module>
    pl.seed_everything(conf.program.seed)
  File "/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/lightning/fabric/utilities/seed.py", line 47, in seed_everything
    seed = int(seed)
ValueError: invalid literal for int() with base 10: '1, 2'
slurmstepd: error: *** JOB 8997578 ON ccc0284 CANCELLED AT 2023-06-10T01:34:24 ***
